frame/support/src/storage/mod.rs

```diff
@@ -732,4 +783,27 @@ mod test {
 			assert_eq!(Digest::decode(&mut &value[..]).unwrap(), expected);
 		});
 	}
+
+	#[test]
+	#[should_panic(expected = "Require transaction not called within with_transaction")]
+	fn require_transaction_should_panic() {
```

Comment:
Same as for the other test.

Reply:
I have made the check enabled for test as well so this one is good.